R/use_lintr.R

```diff
@@ -0,0 +1,45 @@
+#' Use lintr in your project
+#'
+#' Create a minimal lintr config file as a starting point for customization
+#'
+#' @param path Path to project root, where a `.lintr` file should be created.
+#' If it already exists, an error will be thrown.
+#' @param type What kind of configuration to create?
+#'
+#'  * `minimal` creates a minimal lintr config, based off the default linters ([linters_with_defaults()]).
```

Comment:
maybe "minimal" should be "tidyverse"? e.g. we might add `type = "google"` or `type = "bioconductor"` here eventually, then "tidyverse" would make more sense by contrast